react/utils/shouldShowSponsoredBadge.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Product,
+  SponsoredBadgePosition,
+} from 'vtex.product-summary-context/react/ProductSummaryTypes'
+
+const shouldShowSponsoredBadge = (
+  product: Product,
+  position: SponsoredBadgePosition,
+  eligiblePositions: SponsoredBadgePosition[]
```

Comment:
Is there a scenario where we need this to be an array? Considering the places where you used this function, wouldn't it be simpler for this parameter to be a string with the type `SponsoredBadgePosition`?

Reply:
You're right. I left it like this because, initially, we had multiple values for the same "macro" position (for instance, `titleBegin` and `titleTop`).

But, since there's no need for that, I'll remove now and, if we add new values, I'll change back.

Reply:
Done https://github.com/vtex-apps/product-summary/pull/390/commits/8e06157d099d999464884ae31af4e5660d2d1ac3